_faqs/faq-12.md

```diff
@@ -1,8 +1,27 @@
 ---
-question: I want to cite RIOT. Which reference should I use?
+question: How does RIOT compare to XYZ?
 ---
 
-Please consider the following references for citation.
+Compared to **vendor specific solutions**, RIOT runs on many different CPU architectures by many different vendors, from a tiny 8-bit MCU with 2 KiB RAM and 32 KiB flash all the way up to high performance ARM Cortex M and RISC-V MCUs.
+Because RIOT abstracts the hardware differences by providing the same API for the same functionality across this large range of MCUs, projects building upon RIOT are highly portable and therefore resilient to changes in the supply chain.
 
-  * Emmanuel Baccelli, Cenk G&uuml;ndo&#287;an, Oliver Hahm, Peter Kietzmann, Martine Lenders, Hauke Petersen, Kaspar Schleiser, Thomas C. Schmidt, Matthias W&auml;hlisch, <a href="{{ "assets/pdfs/riot-ieeeiotjournal-2018.pdf" | relative_url }}">RIOT: An Open Source Operating System for Low-End Embedded Devices in the IoT</a>, IEEE Internet of Things Journal, Vol. 5, No. 6, pp. 4428-4440, December 2018.
+Compared to **Arduino**, RIOT provides advanced features such as native multi-threading and a mature network stack, while still providing a gentle learning curve.
+This allows new users to quickly get productive with RIOT without limiting developers to specific design patterns and software architectures.
+As a nice treat for users migrating from Arduino, our [Arduino compatibility layer](https://doc.riot-os.org/group__sys__arduino.html) allows re-using sketches and libraries from the Arduino world.
 
+Compared to **zephyr**, RIOT is less complex and has a smaller code base.
+This results in a more gentle learning curve and a shorter on-boarding time needed to get productive with RIOT.
+In addition RIOT has a smaller memory footprint and has been ported to 8-bit and 16-bit platforms, allowing RIOT to be used on MCUs that are too resource constraint to run zephyr.
+Zephyr's larger code base and development community does give zephyr an edge when it comes to the number of features supported.
+We do believe that RIOT has an edge over zephyr when a gentle learning curve, memory consumption, or quick prototyping cycles are priorities.
```

Comment:
One might add that RIOT is more focused on the spirit of Free Software, eg. using a Free implementation of nRF's radio as opposed to using the nonfree Softdevice. While I'd love to see that emphasized more (some say RIOT is the Linux of embedded programming, I think it's more the Debian of it), I'm unsure as to whether we really hold ourselves to those standards (apart from applying LGPL), given that we don't have a clear separation of which our boards do actually use nonfree components.

Reply:
> One might add that RIOT is more focused on the spirit of Free Software, eg. using a Free implementation of nRF's radio as opposed to using the nonfree Softdevice. While I'd love to see that emphasized more (some say RIOT is the Linux of embedded programming, I think it's more the Debian of it), I'm unsure as to whether we really hold ourselves to those standards (apart from applying LGPL), given that we don't have a clear separation of which our boards do actually use nonfree components.

This also shows in the difference in governance model between Zephyr and RIOT. Zephyr is supported by 'members', which are among others big non-FOSS companies such as Meta and Google. Those member organizations deliver governing board members and members of the Technical Steering Committee. Those are the decision making institutes at the project (see this [Zephyr Project Charter](https://www.zephyrproject.org/wp-content/uploads/sites/38/2020/09/CLEAN-LF-Zephyr-Charter-20200624-effective-20200901.pdf)). Zephyr states to have vendor-neutral governance (see page 5 of this [2023 overview](https://zephyrproject.org/wp-content/uploads/sites/38/2024/01/Zephyr-Overview-20240110.pdf)).

Nevertheless, RIOT is developed and steered by the grassroots community, having no explicit company behind the project. And thus having no vendors at all, if I understand correctly. This is also a reason for some people to choose using RIOT instead of Zephyr. 

Might be good to explain this difference in governance also shortly.

Reply:
@chrysn, @jkarinkl: I think I addressed this one and will hit resolve on this conversion. Please "unresolve" if you disagree.